app/models/enrollment.rb

```diff
@@ -3,7 +3,7 @@ class Enrollment < ApplicationRecord
 
   def self.list_data
     enrollments = all.map do |enrollment|
-      { id: enrollment.id, location: enrollment.location, schedule: enrollment.schedule, student_limit: enrollment.student_limit, students: enrollment.students.length }
+      { id: enrollment.id, location: enrollment.location, date: enrollment.schedule.in_time_zone("Mountain Time (US & Canada)").strftime('%m/%d/%Y'), time: enrollment.schedule.in_time_zone("Mountain Time (US & Canada)").strftime('%I:%M %p'), student_limit: enrollment.student_limit, students: enrollment.students.length }
```

Comment:
I found a way to test strftime that might be easier for these scenarios. Will show you later.